packages/rest/src/index.ts

```diff
@@ -20,6 +20,7 @@ export * from './rest.server';
 export * from './sequence';
 export * from './rest-http-error';
 export * from './parse-json';
+export * from './validation/request-body.validator';
```

Comment:
Is it a good idea to add `request-body.validator` to our public API? It will make it difficult to evolve its API in the future. 

IIUC, you are trying to access `RequestBodyValidationOptions`. Can we perhaps move definition of that interface to `src/types.ts` instead?